src/hydrabus/hydrabus_mode_spi.h

```diff
@@ -26,5 +26,7 @@ enum {
 	SPI_LSB_FIRST,
 };
 
-#endif /* _HYDRABUS_MODE_SPI_H_ */
+#define SPEED_NB (8)
```

Comment:
As it is required to export that it is better to rename it SPI_SPEED_NB to avoid potential conflict in future